config/enabled.yml

```diff
@@ -1478,3 +1478,10 @@ Rails/UniqBeforePluck:
 Rails/Validation:
   Description: 'Use validates :attribute, hash of validations.'
   Enabled: true
+
+Rails/KeyWordArgs:
```

Comment:
Also does the rails cop config need to match the class name?  (Rails/KeyWordArgs == class KeyWordArgs)